pkgs/FairwindsOps/gonogo/registry.yaml

```diff
@@ -3,9 +3,33 @@ packages:
   - type: github_release
     repo_owner: FairwindsOps
     repo_name: gonogo
-    description: Tool to evaluate upgrade confidence for Kubernetes cluster addons
-    asset: gonogo_{{trimV .Version}}_{{.OS}}_{{.Arch}}.tar.gz
-    checksum:
-      type: github_release
-      asset: checksums.txt
-      algorithm: sha256
+    description: "[alpha] Tool to evaluate upgrade confidence for Kubernetes cluster addons"
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: semver("<= 0.3.0")
+        asset: gonogo_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+        format: tar.gz
+        checksum:
+          type: github_release
+          asset: checksums.txt
+          algorithm: sha256
+          cosign:
+            opts:
+              - --key
+              - https://artifacts.fairwinds.com/cosign.pub
+              - --signature
+              - https://github.com/FairwindsOps/gonogo/releases/download/{{.Version}}/checksums.txt.sig
+              - --insecure-ignore-tlog
+      - version_constraint: "true"
+        asset: gonogo_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+        format: tar.gz
+        checksum:
+          type: github_release
+          asset: checksums.txt
+          algorithm: sha256
+          cosign:
+            opts:
+              - --key
+              - https://artifacts.fairwinds.com/cosign.pub
+              - --signature
+              - https://github.com/FairwindsOps/gonogo/releases/download/{{.Version}}/checksums.txt.sig
```

Comment:
Made a guess that future releases would no longer need `--insecure-ignore-tlog`.

Reply:
The issue isn't solved at v0.3.1, so I opened an issue.

- https://github.com/FairwindsOps/gonogo/issues/180